### PR TITLE
Bringing sns-testing up to speed w.r.t. its dependencies

### PR DESCRIPTION
**Pull request by aterga**
This PR bumps sns-testing to recent version of the IC mono repo, Quill, DFX, ic-cdk, and ic-wasm. This required adjusting a number of things.

### CODE REVIEW

**Comment by aterga** [comment]
Blocked on https://github.com/dfinity/ic/pull/1755

**Comment by dskloetd** [comment]
Will you make the same change in snsdemo?

**Review by anchpop — APPROVED** [review]

File: participate_sns_swap.sh [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-set -euo pipefail
+set -xeuo pipefail
```
Comment (aterga) [inline participate_sns_swap.sh:3]:
```suggestion
set -euo pipefail
```

File: run_basic_scenario.sh [hunk 2]
```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-set -euo pipefail
+set -xeuo pipefail
```
Comment (aterga) [inline run_basic_scenario.sh:3]:
```suggestion
set -euo pipefail
```

**Pull request merged**